config/management/src/lib.rs

```diff
@@ -0,0 +1,212 @@
+                constants::FULLNODE_NETWORK_KEY,
+            );
+            Self::write_key(storage.as_ref(), &mut buffer, constants::OWNER_KEY);
+            Self::write_key(storage.as_ref(), &mut buffer, constants::OPERATOR_KEY);
+            Self::write_key(
+                storage.as_ref(),
+                &mut buffer,
+                constants::VALIDATOR_NETWORK_KEY,
+            );
+
+            writeln!(buffer, "=================================================").unwrap();
+            writeln!(buffer, "Data").unwrap();
+            writeln!(buffer, "=================================================").unwrap();
+
+            Self::write_u64(storage.as_ref(), &mut buffer, constants::EPOCH);
+            Self::write_u64(storage.as_ref(), &mut buffer, constants::LAST_VOTED_ROUND);
+            Self::write_u64(storage.as_ref(), &mut buffer, constants::PREFERRED_ROUND);
+            Self::write_waypoint(storage.as_ref(), &mut buffer, constants::WAYPOINT);
+
+            writeln!(buffer, "=================================================").unwrap();
+
+            buffer
+        } else {
+            panic!("Expected Command::Verify");
+        }
+    }
+
+    fn write_key(storage: &dyn Storage, buffer: &mut String, key: &str) {
+        let value = storage
+            .get_public_key(key)
+            .map(|c| c.public_key.to_string())
+            .unwrap_or_else(|_| "None".into());
+        writeln!(buffer, "{} - {}", key, value).unwrap();
+    }
+
+    fn write_u64(storage: &dyn Storage, buffer: &mut String, key: &str) {
+        let value = storage
+            .get(key)
+            .and_then(|c| c.value.u64())
+            .map(|c| c.to_string())
+            .unwrap_or_else(|_| "None".into());
+        writeln!(buffer, "{} - {}", key, value).unwrap();
+    }
+
+    fn write_waypoint(storage: &dyn Storage, buffer: &mut String, key: &str) {
+        let value = if let Ok(value) = storage.get(key).and_then(|c| c.value.string()) {
+            if value.is_empty() {
+                "empty".into()
+            } else {
+                Waypoint::from_str(&value)
+                    .map(|c| c.to_string())
+                    .unwrap_or_else(|_| "Invalid waypoint".into())
+            }
+        } else {
+            "None".into()
+        };
+
+        writeln!(buffer, "{} - {}", key, value).unwrap();
+    }
+}
+
+#[derive(Debug, StructOpt)]
+pub struct SecureBackend {
+    #[structopt(long)]
+    host: String,
+    #[structopt(long)]
+    token: String,
+    #[structopt(long)]
+    namespace: Option<String>,
+}
+
+fn secure_storage(config: SecureBackend) -> Box<dyn Storage> {
+    Box::new(VaultStorage::new(
+        config.host,
+        config.token,
+        config.namespace,
+    ))
+}
+
+#[cfg(test)]
+pub mod tests {
+    use super::*;
+    use libra_secure_storage::{Policy, Value};
+
+    const VAULT_HOST: &str = "http://localhost:8200";
+    const VAULT_ROOT_TOKEN: &str = "root_token";
+
+    #[test]
+    #[ignore]
```

Comment:
Nit: maybe add a comment that these tests require the docker script to be running in `docker/vault/run.sh`?

Reply:
I have a good chance now that bors isn't listening.